mlir/lib/Target/LLVMIR/ModuleTranslation.cpp

```diff
@@ -1758,6 +1758,48 @@ ModuleTranslation::convertParameterAttrs(LLVMFuncOp func, int argIdx,
   return attrBuilder;
 }
 
+LogicalResult ModuleTranslation::convertArgAndResultAttrs(
+    ArgAndResultAttrsOpInterface attrsOp, llvm::CallBase *call,
+    ArrayRef<unsigned> immArgPositions) {
+  // Convert the argument attributes.
+  if (ArrayAttr argAttrsArray = attrsOp.getArgAttrsAttr()) {
+    unsigned argAttrIdx = 0;
+    llvm::DenseSet<unsigned> immArgPositionsSet(immArgPositions.begin(),
```

Comment:
nit: For consistency with `convertArgAndResultAttrs`
```suggestion
    llvm::SmallDenseSet<unsigned> immArgPositionsSet(immArgPositions.begin(),
```